src/server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -0,0 +1,158 @@
+/*
+  Copyright 2018 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+import '@polymer/paper-button/paper-button';
+import '@polymer/paper-progress/paper-progress';
+import '@polymer/iron-icon/iron-icon';
+import '@polymer/iron-icons/iron-icons';
+import './outline-step-view';
+
+import {css, customElement, html, LitElement, property, unsafeCSS} from 'lit-element';
+
+import {styleElement} from './cloud-install-styles';
+
+export interface Location {
+  id: string;
+  name: string;
+  flag: string;
+  available: boolean;
+}
+
+@customElement('outline-region-picker-step')
+export class OutlineRegionPicker extends LitElement {
+  @property({type: Array}) locations: Location[] = [];
+  @property({type: String}) selectedLocationId: string = null;
+  @property({type: Boolean}) isServerBeingCreated = false;
+  @property({type: Function}) localize: Function;
+
+  static get styles() {
+    const styles = styleElement.querySelector('template').content.textContent;
+    return css`
+      ${unsafeCSS(styles)}
```

Comment:
Can you export a css`` constant in cloud-install-styles instead so we can use it here and elsewhere?

Thanks for helping to figure this out. We'll reuse it in future components!